format/spec.md

```diff
@@ -513,6 +514,17 @@ Manifests for a snapshot are tracked by a manifest list.
 
 Valid snapshots are stored as a list in table metadata. For serialization, see Appendix C.
 
+Statistics files' metadata within `statistics` field is a struct with the following fields:
+
+| Field name                      | Type                               | Description                                                                                                                                                                                                              |
+|---------------------------------|------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`location`**                  | `string`                           | Location of the statistics file. See [Puffin file format](../puffin).                                                                                                                                                    |
```

Comment:
Other places in the spec use "path" rather than "location" for files. How about using `stats-path` or `statistics-path`?

Also, this is out of date. It should link to `puffin-spec`.

Reply:
changed, thanks